boards/arm/sensortile_box/doc/index.rst

```diff
@@ -0,0 +1,206 @@
+  USB CDC class.
+
+Console default settings are 115200 8N1.
+
+USB interface
+=============
+
+SensorTile.box can be connected as a USB device to a PC host through its micro-B connector.
+The final application may use it to declare SensorTile.box device as belonging to a
+certain standard or vendor class, e.g. a CDC, a mass storage or a composite device with both
+functions.
+
+Programming and Debugging
+*************************
+
+There are 2 main entry points for flashing STM32FL4Rx SoCs, one using the ROM
+bootloader, and another by using the SWD debug port (which requires additional
+hardware) on FTSH107 connector.
+Flash using the ROM bootloader by powering on the board
+while keeping the BOOT0 button pressed.
+The ROM bootloader supports flashing via USB (DFU), UART, I2C and SPI.
+You can read more about how to enable and use the ROM bootloader by checking
+the application note `AN2606`_ (STM32L4Rx section).
+
+Flashing
+========
+
+Installing dfu-util
+-------------------
+
+It is recommended to use at least v0.8 of dfu-util. The package available in
+Debian and Ubuntu can be quite old, so you might have to build dfu-util from source.
+Information about how to get the source code and how to build it can be found
+at the `DFU-UTIL website`_
+
+Flashing an Application to SensorTile.box
+-----------------------------------------
+
+While pressing the BOOT0 button, connect the
+micro-USB cable to the USB OTG SensorTile.box
+port and to your computer. The board should be
+forced to enter DFU mode.
+
+Confirm that the board is in DFU mode:
+
+.. code-block:: console
+
+   $ sudo dfu-util -l
+   dfu-util 0.9
+
+   Copyright 2005-2009 Weston Schmidt, Harald Welte and OpenMoko Inc.
+   Copyright 2010-2019 Tormod Volden and Stefan Schmidt
+   This program is Free Software and has ABSOLUTELY NO WARRANTY
+   Please report bugs to http://sourceforge.net/p/dfu-util/tickets/
+
+   Found DFU: [0483:df11] ver=2200, devnum=74, cfg=1, intf=0, path="2-2", alt=2, name="@OTP Memory /0x1FFF7000/01*0001Ke", serial="204A325D574D"
+   Found DFU: [0483:df11] ver=2200, devnum=74, cfg=1, intf=0, path="2-2", alt=1, name="@Option Bytes  /0x1FF00000/01*040 e/0x1FF01000/01*040 e", serial="204A325D574D"
+   Found DFU: [0483:df11] ver=2200, devnum=74, cfg=1, intf=0, path="2-2", alt=0, name="@Internal Flash  /0x08000000/512*0004Kg", serial="204A325D574D"
+
+You should see following confirmation on your Linux host:
+
+.. code-block:: console
+
+   $ dmesg
+   usb 2-2: new full-speed USB device number 74 using xhci_hcd
+   usb 2-2: New USB device found, idVendor=0483, idProduct=df11
+   usb 2-2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+   usb 2-2: Product: STM32  BOOTLOADER
+   usb 2-2: Manufacturer: STMicroelectronics
+   usb 2-2: SerialNumber: 204A325D574D
+
+You can build and flash the provided sample application
+(:ref:`sensortile_box sensors`) that reads sensors data and outputs
+values on the console.
+
```

Comment:
Since you provided the dfu-runner info, I guess that usual west flash command will work.
It would be good adding and example of west flash/debug commands for zephyr beginners.